src/main/java/org/torproject/onionoo/docs/SummaryDocument.java

```diff
@@ -309,6 +309,28 @@ public String getHostName() {
     return this.hostName;
   }
 
+  @JsonProperty("vh")
```

Comment:
Hmm, hmm, hmm, do we really want to include these new fields in summary documents, too? Maybe, I didn't think too hard about this yet. What do you think, should this change be here or not? (Truly open question.)

Reply:
They need to be here to allow for filtering based on them.

Reply:
Alright, makes sense. (I didn't look at this code in a while.)